### ISSUE TITLE
Batch Normalisation

### ISSUE THREAD

**Issue by kannan500** [jiecaoyu/XNOR-Net-PyTorch#91]
Where in the code,are you using Batch Norm?How its done??

**Comment by jiecaoyu** [comment]
@kannan500 Yes, I am using batch norm. Please check the original paper for more details.
